tox.ini

```diff
@@ -1,67 +1,68 @@
 [tox]
+# skip install to avoid rebuilding the world, do the install with setup.py
+# instead of pip below
+skip_install = True
 # these are the environments that will run when you
 # execute `tox` in the command-line
-# Note: linting is not included by default
 envlist =
     lint
+    build
     all-deps
     min-deps
     examples
-
-
-[testenv:all-deps]
-# skip install to avoid rebuilding the world, do the install with setup.py
-# instead of pip below
-skip_install = True
 passenv =
     RASCALINE_BUILD_TYPE
     RUSTUP_HOME
     CARGO*
 
+[testenv:all-deps]
 deps =
     discover
     numpy
     ase
     chemfiles
-
 commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
 [testenv:min-deps]
-skip_install = True
-passenv =
-    RASCALINE_BUILD_TYPE
-    RUSTUP_HOME
-    CARGO*
```

Comment:
`RASCALINE_BUILD_TYPE` is required to be able to run the python tests in either debug or release mode. 

This repository has been initially built to have cargo as the main test runner for everything, and in particular running `cargo test --all` in the root will run [tox and all the python tests](https://github.com/Luthaf/rascaline/blob/master/rascaline-c-api/tests/python-api.rs). `RASCALINE_BUILD_TYPE` is used to pass the information of "build in debug/release mode" from cargo back to cargo through tox => setup.py => cmake => cargo. This is a bit hacky and we could have a discussion on the best way to organize all test & pick a main test runner, but I think this needs to stay around for now.

---

The two other variables are requires in case someone has a non-standard setup for rustup/cargo installation.

Reply:
I thought one could it pass globally. But this doesn't seem the case...